tests/e2e/Makefile

```diff
@@ -15,9 +15,16 @@ SKIP_ES_EXTERNAL ?= false
 export SKIP_KAFKA ?= false
 export E2E_TESTS_TIMEOUT ?= 330
 
+#Skip prepare-e2e-tests and load-assert-job when tests are run on Prow CI
+PROW_CI ?= false
```

Comment:
I would avoid try to avoid putting this into the Makefile, so far there is no knowledge in the makefile about the test infra. 

The docker image for prow in this PR can prepare tests for prow or the prow config could actually remove the jobs that do not run on prow.

Reply:
Ok, I'll do the cleanup using the script that executes the tests in Prow. Updated and removed the changes.